spec/cucumber/formatter/interceptor_spec.rb

```diff
@@ -61,9 +61,9 @@ module Cucumber::Formatter
       end
 
       it 'raises an ArgumentError if it wasn\'t passed :stderr/:stdout' do
-        expect {
+        expect do
           Interceptor::Pipe.unwrap!(:nonsense)
-        }.to raise_error(ArgumentError)
+        end.to raise_error(ArgumentError)
```

Comment:
And, this too. You see what I'm trying to achieve, I hope.

Reply:
Yeah, I totally do. The project looks like it may need a larger rspec update to bring it up to speed stylistically. Thanks!

Reply:
All set here!